### PR TITLE
[Runtime Security]Add and fix xrefs

### PR DESCRIPTION
**Pull request by rcaballeromx**

### CODE REVIEW

File: docs/en/enterprise-edition/content-collections/get-started/get-started.adoc
Comment (jenjoe22) [inline docs/en/enterprise-edition/content-collections/get-started/get-started.adoc:1]: @rcaballeromx Matangi asked me to change the TM symbol to R in the Get Started topic. Please revert this change.

Comment (rcaballeromx) [reply docs/en/enterprise-edition/content-collections/get-started/get-started.adoc:1]: Done
